lpr.Data/AccountData.cs

```diff
@@ -18,7 +18,7 @@ public class AccountData : IAccountData {
       return await _ctx.Account.Where(a => a.GithubId == githubId).FirstOrDefaultAsync();
     }
 
-    public async Task<Account> RegisterGithubAccount(GithubUser githubUser) {
+    public Account RegisterGithubAccount(GithubUser githubUser) {
```

Comment:
Has EF 6 been changed to use '_ctx.SaveChanges()' by default instead of '_csx.SaveChangesAsync()'?
SaveChangesAsync didn't show in my intellisense

Reply:
Oke, This is something we need to look at. Idk at the moment